configs/NYX405V1/config.h

```diff
@@ -0,0 +1,107 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU STM32F405
+
+#define BOARD_NAME NYX405V1
+#define MANUFACTURER_ID NYXT
+
+#define USE_ACC
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_MAG
+
+#define BEEPER_PIN PC13
+#define MOTOR1_PIN PB1
+#define MOTOR2_PIN PB0
+#define MOTOR3_PIN PA3
+#define MOTOR4_PIN PA2
+#define RX_PPM_PIN PA15
+#define SERVO1_PIN PB5
+#define LED_STRIP_PIN PC8
+#define UART1_TX_PIN PA9
+#define UART3_TX_PIN PB10
+#define UART4_TX_PIN PA0
+#define UART6_TX_PIN PC6
+#define UART1_RX_PIN PA10
+#define UART3_RX_PIN PB11
+#define UART4_RX_PIN PA1
+#define UART5_RX_PIN PD2
+#define UART6_RX_PIN PC7
+#define I2C1_SCL_PIN PB6
+#define I2C1_SDA_PIN PB7
+#define LED0_PIN PA8
+#define SPI1_SCK_PIN PA5
+#define SPI2_SCK_PIN PB13
+#define SPI3_SCK_PIN PC10
+#define SPI1_SDI_PIN PA6
+#define SPI2_SDI_PIN PB14
+#define SPI3_SDI_PIN PC11
+#define SPI1_SDO_PIN PA7
+#define SPI2_SDO_PIN PB15
+#define SPI3_SDO_PIN PC12
+#define ADC_VBAT_PIN PC4
+#define ADC_RSSI_PIN PC5
+#define ADC_CURR_PIN PC3
+#define FLASH_CS_PIN PB12
+#define MAX7456_SPI_CS_PIN PA4
+#define GYRO_1_EXTI_PIN PB8
+#define GYRO_1_CS_PIN PB3
+
+#define TIMER_PIN_MAPPING                                                      \
+  TIMER_PIN_MAP(0, PB1, 1, 0)                                                  \
+  TIMER_PIN_MAP(1, PB0, 1, 0)                                                  \
+  TIMER_PIN_MAP(2, PA2, 1, 0)                                                  \
+  TIMER_PIN_MAP(3, PA3, 1, 1)                                                  \
+  TIMER_PIN_MAP(4, PC8, 2, 0)                                                  \
+  TIMER_PIN_MAP(5, PA15, 1, 0)                                                 \
+  TIMER_PIN_MAP(6, PB5, 1, 0)
+
+#define ADC1_DMA_OPT        1
+#define MAG_I2C_INSTANCE I2CDEV_1
+#define BARO_I2C_INSTANCE I2CDEV_1
+#define DEFAULT_BARO_I2C_ADDRESS 119
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_ON
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 240
+#define DEFAULT_VOLTAGE_METER_SCALE 110
+#define BEEPER_INVERTED
+#define USE_FLASH
+#define FLASH_SPI_INSTANCE SPI2
+#define SYSTEM_HSE_MHZ 8
+#define MAX7456_SPI_INSTANCE SPI1
+#define GYRO_1_SPI_INSTANCE SPI3
+#define GYRO_1_ALIGN CW90_DEG
+#define DEFAULT_FEATURES (FEATURE_OSD | FEATURE_LED_STRIP | FEATURE_GPS | FEATURE_ESC_SENSOR )
+
+#define SERIALRX_UART SERIAL_PORT_USART1
```

Comment:
UARTs 1 & 3 are susceptible to DFU hijacking and should not be used for receivers or GPS.
```suggestion

```